website/content/docs/setup/quickstart/index.md

```diff
@@ -7,9 +7,11 @@ This guide lets you quickly evaluate {{< name >}}. For a custom setup follow the
 
 ## Before you begin
 Ensure that you have the following:
-- `kubectl` with access to a Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
 - `git`
 - `make`
+- `go` version 1.13 and above
```

Comment:
Do you know what requires go? Maybe we can easily drop this requirement with little changes

Reply:
I agree. Go should be a dependency for development but not if you just want to deploy.

Reply:
ok, will remove

Reply:
because it's not really needed?

Reply:
its' not needed (now that I remove the `make install-tools`)